test/scripts/e2e.sh

```diff
@@ -117,23 +118,27 @@ export GOPATH=$(go env GOPATH)
 cd "${SCRIPT_PATH}"
 
 if [ -z "$E2E_TEST_FILTER" ] || [ "$E2E_TEST_FILTER" == "SCRIPTS" ]; then
-    ./timeout 200 ./e2e_basic_start_stop.sh
-    duration "e2e_basic_start_stop.sh"
-
     python3 -m venv "${TEMPDIR}/ve"
     . "${TEMPDIR}/ve/bin/activate"
     "${TEMPDIR}/ve/bin/pip3" install --upgrade pip
     "${TEMPDIR}/ve/bin/pip3" install --upgrade py-algorand-sdk cryptography
     duration "e2e client setup"
 
     if [ $INTERACTIVE ]; then
-        echo "********** READY **********"
-        echo "The test environment is now set. Run the tests using the following command on a different terminal after setting the path."
-        echo ""
-        echo "export VIRTUAL_ENV=\"${TEMPDIR}/ve\""
-        echo "export PATH=\"\$VIRTUAL_ENV/bin:\$PATH\""
+        echo -e "\n\n********** READY **********\n\n"
+        echo "The test environment is now set. You can now run tests in another terminal."
+
+        echo -e "\nConfigure the environment:\n"
+        if [ "$(basename $SHELL)" == "fish" ]; then
+            echo "set -g VIRTUAL_ENV \"${TEMPDIR}/ve\""
+            echo "set -g PATH \"\$VIRTUAL_ENV/bin:\$PATH\""
+        else
+            echo "export VIRTUAL_ENV=\"${TEMPDIR}/ve\""
+            echo "export PATH=\"\$VIRTUAL_ENV/bin:\$PATH\""
+        fi
+
         echo ""
-        echo "${TEMPDIR}/ve/bin/python3" test/scripts/e2e_client_runner.py ${RUN_KMD_WITH_UNSAFE_SCRYPT} "$SRCROOT"/test/scripts/e2e_subs/SCRIPT_FILE_NAME
+        echo "python3 \"$SCRIPT_PATH\"/e2e_client_runner.py ${RUN_KMD_WITH_UNSAFE_SCRYPT} \"$SCRIPT_PATH\"/e2e_subs/SCRIPT_FILE_NAME"
```

Comment:
I am concerned about using python3 form the user environment instead of using python3 from the ve.
I had issues with that.
Is there a downside of using ve/bin/python3 instead of $PATH/python3?

Reply:
We definitely want to use python3 from the virtual environment. The environment variable should make sure we get the correct version: `$VIRTUAL_ENV/bin:$PATH`

Reply:
However, when you open a different terminal per the instructions above, the path will not be set there. And therefore, the standard system python3 will be executed, and not the ve/bin/python3.
Python ve once activated, has a funny way of setting the path.

Reply:
Never mind. What you have is correct. I missed the path part in the guildlines.